examples/async-message-sender.php

```diff
@@ -9,7 +9,26 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-$publisher = new \Burrow\RabbitMQ\AmqpAsyncPublisher('127.0.0.1', 5672, 'guest', 'guest', 'xchange');
+$host = '127.0.0.1';
+$port = 5672;
+$user = 'guest';
+$pass = 'guest';
+$exchange = 'xchange';
+
+//$publisher = new \Burrow\RabbitMQ\AmqpAsyncPublisher($host, $port, $user, $pass, $exchange);
+
+// $connection = new \PhpAmqpLib\Connection\AMQPStreamConnection($host, $port, $user, $pass);
+// $channel = $connection->channel();
+// $messagePublisher = new \Swarrot\Broker\MessagePublisher\PhpAmqpLibMessagePublisher($channel, $exchange);
+
+$credentials = array('host' => $host, 'port' => $port, 'vhost' => '/', 'login' => $user, 'password' => $pass);
+$connection = new \AMQPConnection($credentials);

```

Comment:
Yep.

If you want to use the swarrot implementation of burrow, you'll have to build your swarrot object with the implementation you want.

I might create a factory to retrieve the swarrot objetcs, but that won't be part of that lib.